recipients_app/lib/view/pages/dashboard_page.dart

```diff
@@ -41,8 +50,26 @@ class _DashboardView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final List<DashboardCard> dashboardItems =
-        context.watch<DashboardCardManagerCubit>().state.cards;
+    final List<DashboardItem> dashboardItems = context
```

Comment:
This doesnt make sense. It could just be: context.watch<DashboardCardManagerCubit>().state.cards;

Reply:
It's done to have both types of cards on dashboard. So one cubit return DashboardCards and second returns SurveyCardContainer, both of those widgets are children of DashboardItem to make sure that we can have it in one collection.